### PR TITLE
{AzurePipelineTasks} fixes Azure/azure-powershell#19209

### PR DESCRIPTION
**Pull request by navba-MSFT**
fixes Azure/azure-powershell#19209

Customers have reported an issue with the Pipeline tasks for while it tries to fetch the storage account details it fails with below error:

##[error]Storage account: <my existing and accessible storage account> not found. The selected service connection 'Service Principal' supports storage accounts of Azure Resource Manager type only.

Cause of the issue:
Get-AzureRmStorageAccount is outdated and needs to be replaced with Get-AzStorageAccount.

**Task name**: <Name of changed or new pipeline task>

**Description**: <Describe your changes here>

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** (Y/N) <Please add link to related issue here>

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Comment by mmrazik** [comment]
Please bump the task version (see the checklist in the PR description). Try to rebase also on latest master.

**Comment by mmrazik** [comment]
Or maybe wait. This will probably need some more work. Is there any specific reason why you are fixing this in V4 of the task?

**Comment by navba-MSFT** [comment]
> Or maybe wait. This will probably need some more work. Is there any specific reason why you are fixing this in V4 of the task?

@mmrazik I just tried to change everywhere the `Get-AzureRmStorageAccount `was referenced, Since that the old PS module and should be changed to Get-AzStorageAccount. So I changed it in both v4 and v5 tasks.

**Review by mmrazik — CHANGES_REQUESTED** [review]
Please bump the version of the task you are changing.

**Comment by v-nagarajku** [comment]
I have completed this PR - https://dev.azure.com/mseng/AzureDevOps/_git/AzureDevOps/pullrequest/663407
for creating AzureFileCopyV5 task.
cc: @mmrazik @PhilipsonJoseph

**Comment by navba-MSFT** [comment]
@v-nagarajku Thanks for reviewing this PR. I see all the checks have been passed. May I know when this PR would be merged  / released ?

**Comment by navba-MSFT** [comment]
@mmrazik @PhilipsonJoseph @v-nagarajku Since all the checks have been passed. May I know when this PR would be merged / released ?

**Comment by PhilipsonJoseph** [comment]
@mmrazik  Yes this changes are applicable for v4 too as there is an issue on fetching storage accounts. 

@navba-MSFT Yes you are good to merge the PR. Before that please resolve the Martin's comment

**Comment by navba-MSFT** [comment]
@PhilipsonJoseph 

> Please bump the version of the task you are changing.

This has been done. Also I am not authorized to merge this PR. Could you please help to merge this PR ?

**Review by mmrazik — APPROVED** [review]

File: Tasks/AzureFileCopyV4/task.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 4,
-        "Minor": 209,
+        "Minor": 210,
```
Comment (nijoshms) [inline Tasks/AzureFileCopyV4/task.json:16]: This PR caused issue; we will revert the task.

**Pull request merged**